src/content/reference/react/Fragment.md

```diff
@@ -173,16 +173,16 @@ function Blog() {
 }
 ```
 
-You can inspect the DOM to verify that there are no wrapper elements around the Fragment children:
+Anda bisa memeriksa DOM untuk memastikan bahwa tidak ada elemen pembungkus di sekitar anak *Fragment*:
```

Comment:
```suggestion
Anda bisa memeriksa DOM untuk memastikan bahwa tidak ada elemen pembungkus di sekitar anak-anak *Fragment*:
```

Children should translate to anak-anak.

Reply:
Also can help a bit more to have `anak-anak (*children*)` once in the beginning so we know the context.